new/LIT6166MiraclesJohnBaptist.xml

```diff
@@ -53,10 +55,170 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     </teiHeader>
     <text>
         <body>
+          <div type="edition">
+            <note>The text follows <ref type="mss" corresp="ESmy023"/>.</note>
+             <div type="textpart" subtype="chapter" xml:id="Introduction">
+                 <label>Introduction</label>
+                 <ab>
+                   <pb n="78ra"></pb>
+                   በሰመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡
+                   ንዌጥን፡ በረድኤት፡ እግዚአብHኤር፡ ጽሒፈ፡ ተአምራቲሁ፡ ወመንክራቲሁ፡ ለኮከበ፡ ገዳመ፡
+                   ዮርዳኖስ።
+                 </ab>
+              </div>
+
+              <div type="textpart" subtype="chapter" xml:id="Miracle1">
+                 <label>Miracle 1</label>
+                 <ab>
+                   <pb n="78rb"></pb>
+                   በ፩ተአምረሁ፡ ለብፁዕ፡ ነቢይ፡
+                   ወሰማዕት፡ ቅዱስ፡ ዮሐንስ፡
+                    መጥምቅ፡ ዘተኃርየ፡ እምከርሠ፡
+                    እሙ፡ ከመ፡ ኤርምያስ፡ ነቢዩ። ስምዑ፡ ንነግረክሙ፡ አኃዊነ፡ አድ፡ ወአንስት፡ <gap reason="omitted" resp="DN"/>
+                    ወ<pb n="78va"></pb>ነበረ፡ ፩ብእሲ፡ ዘረሰየ፡ ብሔሮ፡ መልዕልተ፡ አሐቲ፡ ደብር፡ ዘምድረ፡ ኅርጉድ፡
+                 </ab>
+              </div>
+
+              <div type="textpart" subtype="chapter" xml:id="Miracle2">
+                <label>Miracle 2</label>
+                <ab>
+                  <pb n="79rb"></pb>
+                  ፪ተአምረሁ፡ ለብፁዕ፡ ወቅዱስ፡ ዮሐንስ፡ መጥምቅ፡ ጸሎቱ፡ የሃሉ፡ ምስለ፡ አግብርቲሁ፡
+                  አቡነ፡ ወልደ፡ ኪዳን፡ ለዓለመ፡ ዓለም፡ አሜን።
+                  ወካዕበ፡ ስምዑ፡ ንንግርክሙ፡ ዘገብረ፡ ሎቱ፡ ተአምረ፡ ለውእቱ፡ ብእሲ፡ ዘአቅደምን፡ ነጊረ፡
+                  በእንቲአሁ። <gap reason="omitted" resp="DN"/> ወሀለውዎ፡ ሰብአ፡ ደም፡
+                  ፅኑዓነ፡ ኃይል፡ እለ፡ ርኁቅ፡ ብሔሮሙ፡
+                </ab>
+              </div>
+
+
+              <div type="textpart" subtype="chapter" xml:id="Miracle3">
+                 <label>Miracle 3</label>
+                 <ab>
+                   <pb n="79vb"></pb>
+                   ፫ተአምረሁ፡ ለብፁዕ፡ ወቅዱስ፡ ዮሐንስ፡ መጥምቅ፡ ጸሎቱ፡ ወበረከቱ፡ የሃሉ፡ አቡነ፡ ዕቁባ፡ ዮሐንስ፡ ወለተ፡ ኢየሱስ፡ ለዓለመ፡ ዓለም፡ አሜን።
```

Comment:
ዕቁባ? mesla missing?
and is there never wa- between the names?

Reply:
a piece is missing, but no wa-